modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -22,7 +22,7 @@ You can use any of the following options for authenticating the Anypoint Service
 Use the username and password authentication option if your users are managed by Anypoint Platform.  
 ** Using <<create-api-ASM-CLI-ConnAps, Connected Apps credentials>>
 +
-Use the xref:access-management::connected-apps-overview.adoc[Connected Apps] authentication option if your users are managed by an external identity provider (IdP).
+Use the xref:access-management::connected-apps-overview.adoc[Connected Apps] authentication option if your users are managed by Anypoint Platform or an external identity provider (IdP).
```

Comment:
Agree that we will also recommend them to use for the case of users of Anypoint Platform, but it is strange to include this change without changing current line 22.